app/src/main/java/com/github/freeman/bootcamp/games/guessit/WaitingRoomActivity.kt

```diff
@@ -363,7 +382,7 @@ fun RoomInfo(
                     horizontalArrangement = Arrangement.SpaceBetween
                 ) {
                     Text(
-                        text = NB_ROUNDS_INFO,
```

Comment:
Why did you remove the companion object and instead put only the string ?
In contrary, we are trying to banish these "flying" strings.